### DISCUSSION TITLE
你好，請問在AnimationComponent設定攻擊animationClip,是在什麼時侯調用的？

### DISCUSSION THREAD

**Discussion by yoshida9911** [m969/EGamePlay#11] [Q&A]
在excel文檔好像沒有技能相應的animationClip 的資料。
想知道 RpgExample𥚃，attack animation動作是在哪時候被調用的，謝謝
謝謝

**Comment by m969** [comment]
RpgExample的几个技能，是在Timeline里配置的动作和时间，程序也是读的Timeline的数据
